unit/util/piped_process.cpp

```diff
@@ -61,6 +66,39 @@ TEST_CASE(
   REQUIRE(response.find(expected_error) < response.length() - 5);
 }
 
+// This is a test of child termination, it's not perfect and could go wrong
+// if run at midnight, but it's sufficient for a basic check for now.
+TEST_CASE(
+  "Creating a sub process and terminate it.",
+  "[core][util][piped_process]")
+{
+  std::vector<std::string> commands;
+#ifdef _WIN32
+  commands.push_back("cmd /c ping 127.0.0.1 -n 6 > nul");
+  SYSTEMTIME st;
+  GetSystemTime(&st);
```

Comment:
:pick: This should probably be done using C++11's `std::chrono::steady_clock` from the `<chrono>` header, so as to avoid unnecessarily making the test operating system specific.

Reply:
Fixed